server/enterprise.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 func (s *Server) triggerEETestsForOrgMembers(pr *model.PullRequest) {
-	if s.IsOrgMember(pr.Username) {
+	if s.IsOrgMember(pr.Username) || s.IsKnownUser(pr.Username) {
```

Comment:
I renamed it to IsTrustedUser(). I am for adding this, because the translation bot only modifies translation files and otherwise I have to add the EETests label and then check in again in an hour. But if you have objections, I am fine with not whitelisting the CLAExcluded bots.

Reply:
No, I'm fine and there's no harm running EE tests anyway.